### ISSUE TITLE
Pro features not showing anymore

### ISSUE THREAD

**Issue by nefelikaranikola** [dudaster/ele-custom-skin#919]
I have a license for the pro version, I created multiple slider loops in this website everything was working and now I'm supposed to put the website live and suddenly the pro options aer not there anymore.

<img width="709" alt="Screenshot 2023-09-12 at 4 36 58 PM" src="https://github.com/dudaster/ele-custom-skin/assets/64328856/f7f29c86-c3da-42b0-b071-4fa7dbbbe125">

**Comment by hotlavamedia** [comment]
Seeing this as well, perhaps it's related to the Elementor/Elementor Pro updates in the last 24 hours?

Update: Rolling back to the previous version of Elementor/Pro restored functionality.

**Comment by joshbnice** [comment]
Yes, I am dealing with the same issue and it is a crucial part of the functionality of my site.

**Comment by mattennant** [comment]
Same issue for me I'm afraid, following

**Comment by nefelikaranikola** [comment]
I figured out too that the problem occured when I updated Elementor and Elementor Pro. I rolled back only the free version not the ro and now it's working again.

**Comment by peteswan** [comment]
Same here guys. Following

**Comment by dudaster** [comment]
I’m on it.

**Comment by malcolmwrites** [comment]
Same here! I run a really big site and now all the displays and custom skins on my home page and posts are messed up. Hopefully it can get resolved by end of day.

**Comment by LinzySN** [comment]
Have these issues been resolved? I have put off updating Elementor Pro because of what seems to be an incompatibility with the current version of Elementor Custom Skin Pro. I have not yet seen an update.

**Comment by micmiller-apptio** [comment]
I think this may have been an issue that was resolved with Elementor 3.16.2. I'm running 3.16.4 on a site and see the ECS Pro features. Version 3.16.2 fixed some deprecation issues impacting third-party plugins that had been introduced with 3.16.0.
